changelog/Microsoft.AAD.Reporting.json

```
@@ -1103,6 +1103,104 @@
+              "ChangeType": "Addition",
+              "Description": "Added the **servicePrincipalCredentialKeyId** property to the [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+              "Target": "signIn"
+          },
+          {
+              "Id": "245233fd-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Property",
+              "ChangedApiName": "servicePrincipalCredentialThumbprint",
+              "ChangeType": "Addition",
+              "Description": "Added the **servicePrincipalCredentialThumbprint** property to the [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+              "Target": "signIn"
+          },
+          {
+              "Id": "2456e2d7-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Resource",
+              "ChangedApiName": "privateLinkDetails",
+              "ChangeType": "Addition",
+              "Description": "Added the [privateLinkDetails](https://docs.microsoft.com/en-us/graph/api/resources/privateLinkDetails?view=graph-rest-beta) resource type.",
+              "Target": "privateLinkDetails"
+          },
+          {
+              "Id": "24594642-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Member",
+              "ChangedApiName": "deviceFilterIncludeRuleNotMatched",
+              "ChangeType": "Addition",
+              "Description": "Added the `deviceFilterIncludeRuleNotMatched` member to the **conditionalAccessRule** enumeration.",
+              "Target": "conditionalAccessRule"
+          },
+          {
+              "Id": "245ba66e-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Member",
+              "ChangedApiName": "allDeviceStates",
+              "ChangeType": "Addition",
+              "Description": "Added the `allDeviceStates` member to the **conditionalAccessRule** enumeration.",
+              "Target": "conditionalAccessRule"
+          },
+          {
+              "Id": "245e0b51-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Enum type",
+              "ChangedApiName": "signInAccessType",
+              "ChangeType": "Addition",
+              "Description": "Added the **signInAccessType** enumeration type.",
+              "Target": "signInAccessType"
+          },
+          {
+              "Id": "24606f6b-cf8c-11eb-995f-5903f88189f5",
+              "ApiChange": "Member",
+              "ChangedApiName": "unknownFutureValue",
+              "ChangeType": "Deletion",
+              "Description": "Removed the `unknownFutureValue` member from the **conditionalAccessRule** enumeration.",
+              "Target": "conditionalAccessRule"
+          }
+      ],
+      "Id": "2443cc12-cf8c-11eb-995f-5903f88189f5",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2021-06-17T16:50:37.572Z",
+      "WorkloadArea": "Reports and audit",
```

Not aware if there's been a change from **Reports**.
```suggestion
      "WorkloadArea": "Reports",
```